client/src/containers/HistoryViewer/HistoryViewerStateRouter.js

```diff
@@ -24,6 +26,7 @@ const historyStateRouter = (RoutedComponent) => {
       return (
         <RoutedComponent
           {...this.props}
+          store={window.ss.store}
```

Comment:
Shouldn't we rely on the connect in `components/HistoryViewer/HistoryViewer.js` only, and not inject the `window.ss.store` global here. I'll leave a note on improving the store connection. :D

Reply:
Actually, can you try `import Config from 'lib/Config';` and importing the section config at the HOC level, and then just inject the schemaURL property here? You don't need to use redux / store at all then.

Reply:
If we need `store={window.ss.store}` for connect to work then I guess leave that in, but don't access `this.props.store` directly in HistoryViewer.js.

Reply:
FYI we can't put the store in `Injector` or expose it via `expose-loader` very easily as far as I can tell. The store is lazily build in an `Injector.ready()` callback, which means it can't really be a singleton. I think a global store object is the only way to share it at this point.

Reply:
I've updated this so that it has `<Provider store={window.ss.store}><HistoryViewer ...props />...` in render() - so far as I can tell at this stage we must use `window.ss.store`, but I've removed the other direct use of the redux store so it's confined to this HoC only now.